packages/amplify-category-auth/resources/overrides-resource/auth/package.json

```diff
@@ -8,7 +8,7 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
-    "@aws-amplify/cli-extensibility-helper": "^2.0.0"
+    "@aws-amplify/cli-extensibility-helper": "2.4.10-rc.e0c735015.0"
```

Comment:
TODO: update this and similar when https://github.com/aws-amplify/amplify-cli/pull/11961 ships.